src/styles/components/_form.scss

```diff
@@ -0,0 +1,217 @@
+input {
+    width: 100%;
+    cursor: pointer;
```

Comment:
This is making all inputs having the cursor pointer behavior, this is not correct, only type="submit" should have this.